packages/drivers/odsp-driver/src/odspDocumentDeltaConnection.ts

```diff
@@ -633,12 +634,20 @@ export class OdspDocumentDeltaConnection extends DocumentDeltaConnection {
 			}
 		});
 
-		await super.initialize(connectMessage, timeout).finally(() => {
+		try {
+			await super.initialize(connectMessage, timeout);
+			if (this._details) {
+				if (!this.details.supportedFeatures) {
+					this._details.supportedFeatures = {};
+				}
+				this._details.supportedFeatures![enable_single_commit_summary] = true;
```

Comment:
`this._details.supportedFeatures![enable_single_commit_summary] = true;` is probably not required since odspDocumentService sets `summarizeProtocolTree` to true. Let me know if that is correct, and I will remove this logic.

Reply:
Yes, for odsp we don't need this in supported features. You can remove this.